intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeaturePluginManager.kt

```diff
@@ -104,8 +104,8 @@ class KtorFeaturePluginManager private constructor(private val extractDirectory:
               usage = doc.usage!!,
               options = doc.options!!
             )
-          }
-
+          },
+          fullDescriptorJson = Json.stringify(KtorFeatureDescriptor.serializer(), descriptor)
```

Comment:
Do you need? JsonConfiguration.Stable.copy(isLenient = true, ignoreUnknownKeys = true)